rails-kickoff-template.rb

```diff
@@ -259,85 +259,169 @@ def set_sentry_context
+    RACK_TIMEOUT_SERVICE_PAST_WAIT
+    ```
 
-    ### Coding Style
+    [rt]: https://github.com/sharpstone/rack-timeout#configuring
 
-    This projects uses RuboCop and ESLint to catch errors and keep style consistent.
+    ### Important rake tasks
 
-    It also uses [lefthook][lh] to manage git hooks. Use `git commit --no-verify` to skip checks, or see [./lefthook.yml](./lefthook.yml) for info on how to change its setup.
+    !TODO: Document important rake taskss
 
-    [lh]: https://github.com/Arkweid/lefthook
+    ### Scheduled tasks
 
+    !TODO: Document any scheduled tasks
 
-    ### Important rake tasks
+    # Deployments
+        
+    ## Staging
 
-    _TODO_
+    [Staging](https://dashboard.heroku.com/apps/!TODO) is automatically 
+    deployed from the `main` branch.
 
-    ### Scheduled tasks
+    ## Production
 
-    _TODO_
+    [Production](https://dashboard.heroku.com/apps/TODO) can be
+    [deployed manually through Heroku's interface](https://dashboard.heroku.com/apps/TODO/deploy/github),
+    or [deployed manually with Git](https://devcenter.heroku.com/articles/git):
 
-    ### Important ENV variables
+    1. Add the appropriate git remotes for staging and/or production deployments:
 
-    Note that this project uses [dotenv](https://github.com/bkeepers/dotenv) to load `.env` files. Use `.env.development` and `.env.test` to setup _shared_ ENV variables for development and test, and use `.env` files ending in `.local` for variables specific to you.
+      ```sh
+      heroku git:remote -r staging -a !TODO-staging
+      heroku git:remote -r production -a !TODO-production
+      ```
 
-    Configuring Servers:
+    2. Push your main branch to production:
 
-    ```
-    WEB_CONCURRENCY - Number of Puma workers
-    RAILS_MAX_THREADS - Number of threads per Puma worker
-    SIDEKIQ_CONCURRENCY - Number of Sidekiq workers
-    ```
+      ```sh
+      git push production main
+      ```
 
-    [rack-timeout][rt]:
+    ## jemalloc
 
-    ```
-    RACK_TIMEOUT_SERVICE_TIMEOUT
-    RACK_TIMEOUT_WAIT_TIMEOUT
-    RACK_TIMEOUT_WAIT_OVERTIME
-    RACK_TIMEOUT_SERVICE_PAST_WAIT
+    This project is served from Heroku. It uses jemalloc to more efficiently
+    allocate memory. You must run the following to setup jemalloc:
+
+    ```bash
+    heroku buildpacks:add --index 1 https://github.com/mojodna/heroku-buildpack-jemalloc.git
     ```
 
-    [rt]: https://github.com/sharpstone/rack-timeout#configuring
+    # Contributing
+
+    ## Branch naming conventions
+
+    Branch names should follow the format: `<ticket_number>-<brief-description>`
+
+    ## Submitting a PR
+
+    - Include the relevant Pivotal issue number(s), when applicable, using the following format:
+
+        ```
+        Delivers [<ticket_number>](https://www.pivotaltracker.com/story/show/<ticket-number>) - <ticket_title> 
+        ```
+
+    !TODO: Add additional guidance, for example: who should be added as reviewers; Any labels/tags needed;
```

Comment:
what do think about moving this into a separate [.github/pull_request_template.md](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) file?

i'd recommend a more simple format, that works with [tracker's github integration](https://www.pivotaltracker.com/help/articles/github_integration/#using-the-github-integration-commits) e.g. 

https://www.pivotaltracker.com/story/show/<ticket-id>
[delivers #<ticket-id>]

i don't think it's totally necessary to include the title

Reply:
I guess the reason I thought including it in the README here is because it's sort of an integral part in jumping into a project and contributing. If I'm new to a project and need to start writing code I need to know: how to spin up the app; where related resources are; how are we handling PRs -- and that last bit includes branch-naming conventions. I guess maybe they should be consistent across projects? But in my experience they haven't been. I don't know if that's because different projects have different preferences, or because I've just been on some odd projects, or maybe folks just don't know?